src/code/z_parameter.c

```diff
@@ -3101,7 +3103,18 @@ void Interface_Draw(PlayState* play) {
         Gfx_SetupDL_39Overlay(play->state.gfxCtx);
 
         // Rupee Icon
+        #ifndef MM_WALLET_ICON_COLORS
```

Comment:
really cool change, how about making a define to let the users choose the color they want? and keep the MM ones by default of course